src/_scss/pages/homepage/features/_covidFeature.scss

```diff
@@ -0,0 +1,244 @@
+.feature-covid {
+    
+    @include media ($medium-screen) {
+        margin-top: -5%;
+    }
+
+    //ROW 1: OFFICIAL SPENDING DATA
+    .official-spending-data__content-wrapper {
+
+        @include media($medium-screen) {
+            @include display(flex);
+            @include justify-content(center);
+        }
+
+        .official-spending-data__text {
+            display: block;
+            margin: rem(-50) rem(10) rem(10) rem(10);
+
+            @include media($medium-screen) {
+                max-width: 40%;
```

Comment:
This will break in IE if we're using max-width. Can we change this to width w/o it breaking chrome etc...?